src/Storages/IStorage.h

```diff
@@ -114,6 +114,9 @@ class IStorage : public std::enable_shared_from_this<IStorage>, public TypePromo
     /// The name of the table.
     StorageID getStorageID() const;
 
+    /// Unique ID, synchronized between replicas for replicated storage
+    virtual String getTableUniqID() const { return ""; }
```

Comment:
Remove, only used for `StorageReplicatedMergeTree`.

Reply:
Done.